src/core/channelz/property_list.h

```diff
@@ -139,7 +141,7 @@ class PropertyList final : public OtherPropertyValue {
   friend class PropertyGrid;
   friend class PropertyTable;
 
-  absl::flat_hash_map<std::string, PropertyValue> property_list_;
+  std::vector<std::pair<std::string, PropertyValue>> property_list_;
```

Comment:
If the goal is to order, can we switch to std::map?

Reply:
The goal is to preserve insertion order

Reply:
Got it.